src/main/java/de/maxhenkel/enhancedgroups/command/AutoJoinGroupCommands.java

```diff
@@ -47,10 +46,14 @@ public int remove(CommandContext<CommandSourceStack> context) throws CommandSynt
     }
 
     public static int autoJoin(CommandContext<CommandSourceStack> context, UUID groupId, @Nullable String password) throws CommandSyntaxException {
+        if (EnhancedGroups.AUTO_JOIN_GROUP_STORE.getGlobalGroupForced()) {
+            context.getSource().sendFailure(Component.literal("Global auto join is enforced on this server"));
+            return 0;
+        }
```

Comment:
Maybe it should still be possible to modify auto joining. The user must just be informed that it won't have any effect because global forced groups are active